contrib/ipp/CMakeLists.txt

```diff
@@ -37,15 +37,77 @@ endif()
 set( CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${ipp_defs}" )
 set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${ipp_defs}" )
 
+### ippinit
+set( sources "" )
```

Comment:
Can we, please, use the same approach as we had here:
https://github.com/Intel-Media-SDK/MediaSDK/blob/master/_studio/mfx_lib/CMakeLists.txt#L131

Please, create OBJECT library from all ipp sources except ippinit.c and link single library, called ipp at the very end.

Reply:
done